Make the navsat tf frame name parametric

ardabbour opened the pull request:
This pull request addresses #698

Commit:
Make the navsat tf frame name parametric

ayrton04 commented on include/robot_localization/navsat_transform.h:
```diff
@@ -165,6 +165,11 @@ class NavSatTransform
+    //! @brief The cartesian frame ID, default as 'local_enu' if using Local Cartesian or 'utm' if using the UTM
+    //! coordinates as our cartesian.
+    //!
+    std::string cartesian_frame_id_;
```
Would you mind putting this down with the other strings? I was aiming for grouping by datatype and _then_ alphabetically, though it appears there are some anomalies here.

ardabbour replied:
done!

ayrton04 reviewed (approved):
Thanks for the PR!

ardabbour committed:
Merge branch 'cra-ros-pkg:noetic-devel' into make-utm-frame-name-parametric

ardabbour committed:
move cartesian_frame_id_

cartesian_frame_id_ was moved to other string types to have header members sorted by datatype first then alphabetically
